### PR TITLE
chore: fixes for talos in docker-compose environment

### PR DESCRIPTION
**Pull request by smira**
* Provides certificate_key (k8s 1.14)

* Fixes local address of master-1, without the fix
it gets default 192.168.x.x assigned when k8s config is updated

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

File: hack/dev/userdata/.master-1.tpl.yaml [hunk 1]
```diff
@@ -2,14 +2,19 @@ services:
...
   kubeadm:
+    certificateKey: verysimplesecretyoucanguess.....
     configuration: |
       apiVersion: kubeadm.k8s.io/v1beta1
       kind: InitConfiguration
       bootstrapTokens:
       - token: 1qbsj9.3oz5hsk6grdfp98b
         ttl: 0s
       nodeRegistration:
+        name: master-1
```
Comment (smira) [inline hack/dev/userdata/.master-1.tpl.yaml]: without it, node hostname gets injected into config so all the containers get the same node name

node name doesn't resolve inside container (separate issue to be fixed), but this brings us one step closer

File: hack/dev/Makefile [hunk 2]
```diff
@@ -29,7 +29,7 @@ kubeconfig:
 .PHONY: clean
 clean: down
 	-@rm talosconfig
-	-@rm kubeconfig
+	-@rm -rf kubeconfig
```
Comment (smira) [inline hack/dev/Makefile]: kubeconfig is a directory, as it's mounted into the volume (I'm not sure if it's what's expected, but this fixes `make clean` to pass without errors)

Comment (andrewrynhard) [reply hack/dev/Makefile]: Not expected, but this bug is annoying so we can keep this.

File: hack/dev/userdata/.master-1.tpl.yaml [hunk 3]
```diff
@@ -2,14 +2,19 @@ services:
   init:
     cni: flannel
   kubeadm:
+    certificateKey: verysimplesecretyoucanguess.....
```
Comment (smira) [inline hack/dev/userdata/.master-1.tpl.yaml]: with #489 this can be changed to 'ChangeMe' as there will be no 32 bytes requrement

**Comment by andrewrynhard** [comment]
@smira I merged a few things. This will need a rebase. 👍

**Comment by smira** [comment]
updated, also fixed one more place with docker image name (#490)

**Pull request merged**